templates/opsportal.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.0-1"
     appversion.kubeaddons.mesosphere.io/opsportal: "1.0.0"
     endpoint.kubeaddons.mesosphere.io/opsportal: /ops/portal/
-    values.chart.helm.kubeaddons.mesosphere.io/opsportal: "https://raw.githubusercontent.com/mesosphere/charts/4155f480571eaf82c64ddd63d3d334b1105d0591/stable/opsportal/values.yaml"
+    values.chart.helm.kubeaddons.mesosphere.io/opsportal: "https://raw.githubusercontent.com/mesosphere/charts/master/stable/opsportal/values.yaml"
```

Comment:
this line ^

Reply:
I don't think we want this pointing to master... I've never changed this value. @shaneutt thoughts?

Reply:
no, shane explained in new repo, we have to update that sha after charts repo has merged 🤷‍♂